bin/get-open-spec.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -ex
+
+VERSIONS="12 13 14 15 16"
+
+rm -rf api_spec
+mkdir api_spec
+for v in 12 13 14 15 16; do
```

Comment:
```suggestion
for v in "${VERSIONS[@]}"; do
```
i think you forgot to use the variable here

Reply:
I did not forget, simply i did not use it. Do you want me to add it?

Reply:
My idea was not to use it